tests/pdfbox/Dockerfile

```diff
@@ -0,0 +1,32 @@
+ARG BASE_IMAGE=ubuntu/chiselled-jre:8_edge
+ARG MAVEN_IMAGE=maven:3.9.0-eclipse-temurin-8
+ARG USER=app
+ARG UID=101
+ARG GROUP=app
+ARG GID=101
+
+FROM $MAVEN_IMAGE as builder
```

Comment:
You can simplify this by removing it, and below, do `COPY --from=$MAVEN_IMAGE`

Reply:
Another podman/docker issue - they do not agree whether to do substitution in --from or not.  ;(